pythonforandroid/bootstraps/sdl2/build/src/main/java/org/kivy/android/PythonActivity.java

```diff
@@ -632,4 +633,12 @@ public void requestPermissionsWithRequestCode(String[] permissions, int requestC
     public void requestPermissions(String[] permissions) {
         requestPermissionsWithRequestCode(permissions, 1);
     }
+
+    public static void changeKeyboard(int inputType) {
+      if (SDLActivity.keyboardInputType != inputType){
+          SDLActivity.keyboardInputType = inputType;
+          InputMethodManager imm = (InputMethodManager) getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
+          imm.restartInput(mTextEdit);
+          }
+    }
```

Comment:
OK nice, then someone could write the Python wrapper in `pythonforandroid/recipes/android/src/android/` later, correct?
If you already have done so for your project you could share the code snipped in some comments. Or better make a follow up PR after this one is merged

Reply:
Yes it could be. 

This change keyboard function is being utilized kivy/kivy/#7231